npm/src/browser.ts

```diff
@@ -111,10 +111,7 @@ export function getCompilerWorker(workerArg: string | Worker): ICompilerWorker {
 }
 
 export async function getLanguageService(): Promise<ILanguageService> {
-  if (!wasmModule) throw "Wasm module must be loaded first";
-  if (!wasmInstancePromise) wasmInstancePromise = initWasm(wasmModule);
-  await wasmInstancePromise;
-
+  await instantiateWasm();
```

Comment:
Is this just a drive by fix-up to match the compiler change? (i.e. did I miss that in my change?)

Reply:
It's to match the compiler change, yes. It _was_ necessary to get logging working though (since logging is initialized within `instantiateWasm`).